packages/cli/src/util/git/connect-git-provider.ts

```diff
@@ -83,15 +92,14 @@ export function formatProvider(type: string): string {
   }
 }
 
-export function parseRepoUrl(originUrl: string): {
-  provider: string;
-  org: string;
-  repo: string;
-} | null {
+export function parseRepoUrl(originUrl: string): ParsedRepoUrl | null {
   const isSSH = originUrl.startsWith('git@');
   // Matches all characters between (// or @) and (.com or .org)
   // eslint-disable-next-line prefer-named-capture-group
-  const provider = /(?<=(\/\/|@)).*(?=(\.com|\.org))/.exec(originUrl);
+  // const provider = /(?<=(\/\/|@)).*(?=(\.com|\.org))/.exec(originUrl);
+  let provider =
```

Comment:
Sorry, this one was sloppy. I initially changed it to `let` because of some other logic I was trying that involved reassigning it, but then I didn't change it back.

Re regex: there's definitely a pattern that will cover everything I want, but I couldn't figure it out :( so I decided it was better to move on for now. If you're good with regex, I would love to pair with you sometime today to figure it out.